content/hi/nodes.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: नोड्स (Nodes) 
-status: Completed
-category: संकल्पना
-exclude_search: true
+शीर्षक: नोड्स (Nodes) 
+स्थिति: समाप्त (completed)
+श्रेणी: संकल्पना
+खोज बहिष्कृत(exclude_search): सही(true)
 ---
 
-## यह क्या है
+# यह क्या है
 
 नोड एक कंप्यूटर है जो एक सामान्य कार्य को पूरा करने के लिए अन्य कंप्यूटरों या नोड्स के साथ मिलकर काम करता है। उदाहरण के लिए, अपना लैपटॉप, मॉडेम और प्रिंटर लें। वे सभी आपके वाई-फाई (Wi-Fi) नेटवर्क से जुड़े रहकर संचार और सहयोग कर रहे हैं और प्रत्येक एक नोड का प्रतिनिधित्व करते है। क्लाउड कंप्यूटिंग में, एक भौतिक कंप्यूटर, एक वर्चुअल कंप्यूटर, (जिसे VM कहा जाता है), या यहां तक ​​कि एक कंटेनर को नोड कहा जा सकता है।
```

Comment:
To Hindi contributors:
[the original English text](https://github.com/cncf/glossary/blob/main/content/en/nodes.md?plain=1) has hyperlinks that go to other terms,
but the Hindi version here does not have any hyperlink.